mypy/main.py

```diff
@@ -1008,6 +1008,12 @@ def add_invertible_flag(
         title="Advanced options", description="Debug and customize mypy internals."
     )
     internals_group.add_argument("--pdb", action="store_true", help="Invoke pdb on fatal error")
+    internals_group.add_argument(
+        "--plugins",
+        nargs="*",
```

Comment:
Using `nargs="*"` might be problematic, since this swallows all of the following positional arguments. For example, the invocation
```shell
mypy --plugins my_plugin src/foo.py
```
will interpret both `my_plugin` and `src/foo.py` are arguments to `plugins`, and will error due to no targets being given. 

Maybe a repeatable argument or a string argument with the same syntax as the config file would work better?